services/catarse/spec/models/user_spec.rb

```diff
@@ -685,4 +685,55 @@ def create_contribution_with_payment(user, project, value, credits, payment_stat
       end
     end
   end
+
+  describe '#before_save' do
+    context 'when user is being created' do
+      context 'when user email domain includes `catarse`' do
+        let(:user) { create(:user, email: 'example@catarse.me') }
+
+        it 'adds error to user mail' do
+          expect(user.errors[:email]).to include(I18n.t('errors.messages.email_not_allowed'))
```

Comment:
**Sugestão:**
Usar a `invalid` na mensagem ao invés do novo texto criado.

Reply:
@stephannv só funciona quando eu modifico:
User::CustomValidators: errors.add(:base, :invalid)
e 
expect(user.errors[:base]).to include(I18n.t('errors.messages.invalid'))
Nesse caso, as mensagens de erro são diferentes e o teste quebra.